internal/command/container.go

```diff
@@ -0,0 +1,393 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/fyne-io/fyne-cross/internal/icon"
+	"github.com/fyne-io/fyne-cross/internal/log"
+	"github.com/fyne-io/fyne-cross/internal/volume"
+	"golang.org/x/sys/execabs"
+)
+
+const (
+	// fyneBin is the path of the fyne binary into the docker image
+	fyneBin = "/usr/local/bin/fyne"
+	// gowindresBin is the path of the gowindres binary into the docker image
+	gowindresBin = "/usr/local/bin/gowindres"
+)
+
+type ContainerRunner interface {
+	NewImageContainer(arch Architecture, OS string, image string) ContainerImage
+
+	Debug(v ...interface{})
+	GetDebug() bool
+}
+
+type AllContainerRunner struct {
```

Comment:
Is this the top level of Fyne-Cross running? (from the content it seems like it might be)?
If so `CrossCompiler` or something high level but specific would be helpful

Reply:
I think that might be the right terminology as we might not always require some form of container/pod for a specific engine, but they will always do some cross compilation.

Reply:
Cool. also, as discussed above today, perhaps `Engine`?